sl/include/platform/clock/nrf_802154_clock.h

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
Like in `driver/`, I would move platform one level above to clearly separate it from headers and implementation